setup.py

```diff
@@ -12,12 +12,12 @@
     packages=find_packages(exclude=['tests', 'example']),
     install_requires=[
         'andi >= 0.4.1',
-        'attrs',
-        'parsel',
-        'url-matcher',
-        'web-poet @ git+https://git@github.com/scrapinghub/web-poet@master#egg=web-poet',
-        'tldextract',
-        'sqlitedict',
+        'attrs >= 21.3.0',
+        'parsel >= 1.5.0',
+        'scrapy >= 2.6.0',
+        'sqlitedict >= 1.5.0',
+        'url-matcher >= 0.2.0',
+        'web-poet >= 0.2.0',
```

Comment:
As per https://github.com/scrapinghub/web-poet/pull/51#issuecomment-1154937042, we should probably make this into `0.3.0`, as well as updating the CHANGELOG and tox.ini.

Reply:
Yes, I went through the same path. It was when I went to apply the contextvar name change to the patch when I realized we do not need 0.3.0 here.